server/graphql/v2/enum/ExpenseProcessAction.ts

```diff
@@ -19,6 +19,9 @@ export const ExpenseProcessAction = new GraphQLEnumType({
     SCHEDULE_FOR_PAYMENT: {
       description: 'To schedule the expense for payment',
     },
+    UNSCHEDULE_PAYMENT: {
+      description: 'To uschedule the expense payment',
+    },
```

Comment:
Why not `UNSCHEDULE_FOR_PAYMENT`?

Reply:
I'm not sure if you can _unschedule for_ something, doesn't sound right. @alanna can you confirm?